app/Http/Controllers/PasswordResetController.php

```
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Password;
+use Illuminate\Support\Facades\Validator;
+
+/**
+ *
+ */
+class PasswordResetController extends Controller
+{
+    /**
+     * Sends a password reset link to the librarian's email.
+     * This endpoint is publicly accessible, but only librarians will receive the email.
+     * If the provided email does not belong to librarian, a 403 error is returned.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function sendResetLinkEmail(Request $request)
+    {
+        $request->validate([
+            'email' => 'required|email'
```

You can check exists validation function from the documentation - [https://laravel.com/docs/12.x/validation#rule-exists]. If there isn’t a Librarian with that email address, we need to return a validation error message.